libs/ledger-live-common/src/families/tezos/bakers.ts

```diff
@@ -31,79 +31,106 @@ export type Delegation = {
   // true if a send should inform it will top down the delegation
   sendShouldWarnDelegation: boolean;
 };
+
+type API_BAKER = {
+  address: string;
+  name: string;
+  logo: string;
+  balance: number;
+  stakingBalance: number;
+  stakingCapacity: number;
+  maxStakingBalance: number;
+  freeSpace: number;
+  fee: number;
+  minDelegation: number;
+  payoutDelay: number;
+  payoutPeriod: number;
+  openForDelegation: true;
+  estimatedRoi: number;
+  serviceType: string;
+  serviceHealth: string;
+  payoutTiming: string;
+  payoutAccuracy: string;
+  audit: string;
+  insuranceCoveragenumber: number;
+};
+
+const getLedgerValidator = async (): Promise<Baker> => {
+  const { data } = await network<API_BAKER>({
```

Comment:
Do we really need to do this call ? Don't we do it for other bakers ?

Reply:
Please do not resolve my comment without any answer

I can find the ledger validator here on this request https://tezos-bakers.api.live.ledger.com/v2/bakers

``` 
...{
    "address": "tz3LV9aGKHDnAZHCtC9SjNtTrKRu678FqSki",
    "name": "Ledger Live by Kiln",
    "logo": "https://services.tzkt.io/v1/avatars/tz3LV9aGKHDnAZHCtC9SjNtTrKRu678FqSki",
    "balance": 708166.216593,
    "stakingBalance": 708166.216593,
    "stakingCapacity": 7081662.16593,
    "maxStakingBalance": 7081662.16593,
    "freeSpace": 6373495.949337,
    "fee": 0.12,
    "minDelegation": 1,
    "payoutDelay": 1,
    "payoutPeriod": 1,
    "openForDelegation": true,
    "estimatedRoi": 0.053786,
    "serviceType": "tezos_only",
    "serviceHealth": "active",
    "payoutTiming": "no_data",
    "payoutAccuracy": "no_data",
    "insuranceCoverage": 0
  },...
  ```

Why the need to call twice the service with a more specific request
https://tezos-bakers.api.live.ledger.com/v2/bakers/tz3LV9aGKHDnAZHCtC9SjNtTrKRu678FqSki

  ```
{
  "address": "tz3LV9aGKHDnAZHCtC9SjNtTrKRu678FqSki",
  "name": "Ledger Live by Kiln",
  "logo": "https://services.tzkt.io/v1/avatars/tz3LV9aGKHDnAZHCtC9SjNtTrKRu678FqSki",
  "balance": 708186.092993,
  "stakingBalance": 708186.092993,
  "stakingCapacity": 7081860.92993,
  "maxStakingBalance": 7081860.92993,
  "freeSpace": 6373674.836936999,
  "fee": 0.12,
  "minDelegation": 1,
  "payoutDelay": 1,
  "payoutPeriod": 1,
  "openForDelegation": true,
  "estimatedRoi": 0.053787,
  "serviceType": "tezos_only",
  "serviceHealth": "active",
  "payoutTiming": "no_data",
  "payoutAccuracy": "no_data",
  "insuranceCoverage": 0,
}
```